spec/forms/teacher_interface/alternative_name_form_spec.rb

```diff
@@ -23,6 +23,54 @@
     let(:alternative_family_name) { "" }
 
     it { is_expected.to be true }
+
```

Comment:
We can refactor these tests in the same way testing the form subject and using the `be_valid` assertion.